app/src/main/AndroidManifest.xml

```diff
@@ -10,14 +10,14 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".SplashScreenActivity">
+        <activity android:name=".ui.AddTaskGoalActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".MainActivity" />
+<!--        <activity android:name=".MainActivity" />-->
```

Comment:
Add Task/Goal became the main activity ?

Reply:
Oups, this file shoudn't be push